internal/pkg/service/buffer/store/configstore/configstore.go

```diff
@@ -49,6 +53,11 @@ func MappingKey(projectID int, receiverID string, exportID string, revisionID in
 	return fmt.Sprintf("config/mapping/revision/%d/%s/%s/%08d", projectID, receiverID, exportID, revisionID)
 }
 
+func RecordKey(projectID int, receiverID string, exportID string, fileID string, sliceID string) string {
+	recordID := time.Now().Format("2006-01-02T15:04:05.000Z07:00") + "_" + idgenerator.Random(5)
```

Comment:
Z [dokumentace](https://pkg.go.dev/time#pkg-constants) mi tak jako přijde, že když do toho layoutu předepíšu ty milisekundy nulama, tak to znamená že je tam zachová a budou se tím pádem klíče řadit správně. (Srovnávám `StampMilli` s nulama oproti `RFC3339Nano` který má v předpisu devítky a dál je výslovně napsaný, že odstraňuje koncový nuly. Ale jako nevím, explicitně to tam napsaný nikde nevidím. 🙂 )

Reply:
jj, je to presne tak `99999` sa odstranuju `00000` nie, preto som to daval aj do dizajn docs.

Reply:
Ta casova zona `07:00` nam moze robit problemy, ukladal by som iba UTC vo formate `2006-01-02T15:04:05.000Z`

Porovnaj: 
https://goplay.tools/snippet/E2OYexD87sS

Idealne ak bude nejaka funkcia `FormatTimeForKey` aby nedoslo k omylu.

Reply:
jo jasně, upraveno